src/resources/js/events-admin.js

```diff
@@ -710,16 +710,53 @@ jQuery( document ).ready( function( $ ) {
 
 });
 
+( function ( $ ) {
+	'use script';
 
-/**
- * Re-initialize chosen on widgets when moved
- * credits: http://www.johngadbois.com/adding-your-own-callbacks-to-wordpress-ajax-requests/
- */
-jQuery( document ).ajaxSuccess( function( e, xhr, settings ) {
-	if ( typeof settings !== 'undefined' && typeof settings.data !== 'undefined' && settings.data.search( 'action=save-widget' ) != - 1 ) {
-		jQuery( "#widgets-right .chosen" ).chosen();
-	}
-} );
+	// Track when the Widget Opens
+	$( document.body ).on( 'click.widgets-toggle', function( e ) {
+		var target = $( e.target ),
```

Comment:
Add a sigil to mark these vars as likely to contain jQuery objects? ie `$target`, `$widget`, `$inside`